app/controllers/user_controller.rb

```diff
@@ -185,6 +185,25 @@ def signup
           end
         end
 
+        # Prevent signups from spam domains
+        if UserSpamScorer.new.email_from_spam_domain?(@user_signup)
+          if send_exception_notifications?
+            msg = "Attempted signup from spam domain email: " \
+                  " #{ @user_signup.email }"
```

Comment:
Don't need the space at the start of this line given the space at the end of the previous line.